README.md

```diff
@@ -161,6 +161,30 @@ func ExampleClient() *redis.Client {
 
 ```
 
+
+### Advanced Configuration
+
+go-redis supports extending the CLIENT SETNAME command to allow for custom client identification. This feature enables community members to append their own suffixes to the client name, facilitating a deeper understanding of Redis usage patterns.
```

Comment:
`go-redis supports extending the client identification phase to allow projects to send their own custom client identification. `